spec/requests/saml_sessions_controller_spec.rb

```diff
@@ -1,13 +1,15 @@
 require "rails_helper"
 require "sidekiq/testing"
 
-RSpec.describe "SamlSessionsController" do
+RSpec.describe SamlSessionsController do
   let(:firm) { create(:firm, offices: [office]) }
   let(:office) { create(:office) }
   let(:provider) { create(:provider, firm:, selected_office: office, offices: [office], username:) }
   let(:username) { "bob the builder" }
-  let(:provider_details_api_url) { "#{Rails.configuration.x.provider_details.url}#{username.gsub(' ', '%20')}" }
-  let(:provider_details_api_reponse) { api_response.to_json }
+  let(:firm_id) { "3959183" }
+  let(:provider_details_api_url) { "#{Rails.configuration.x.pda.url}/provider-users/#{username.gsub(' ', '%20')}/provider-offices" }
+  let(:provider_firms_api_url) { "#{Rails.configuration.x.pda.url}/provider-firms/#{firm_id}/provider-offices" }
+  let(:provider_details_api_response) { api_response.to_json }
```

Comment:
is the api response being used here [this one](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/9e4220d3e8551ed25c880297a320befd249f395f/spec/services/provider_details_creator_spec.rb#L270)? Is that the correct payload to use for the new PDA?

Reply:
No, it should be set in individual test blocks, e.g. 
```ruby 
  let(:api_response) { raw_details_response }
```
Like [here](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/7453/files#diff-589b797e671e70b24d3566c55ac9dda4eacba6297ab202df568cbfd786e6ac33L59)

Reply:
I've checked and removing the linked code does not make these tests fail

Reply:
cool, yes had not noticed the let as it was not part of the changes itself 👍